weave/trace/weave_client.py

```diff
@@ -987,12 +987,13 @@ def create_call(
             attributes = {}
 
         attributes = AttributesDict(**attributes)
-        attributes._set_weave_item("client_version", version.VERSION)
-        attributes._set_weave_item("source", "python-sdk")
-        attributes._set_weave_item("os_name", platform.system())
-        attributes._set_weave_item("os_version", platform.version())
-        attributes._set_weave_item("os_release", platform.release())
-        attributes._set_weave_item("sys_version", sys.version)
+        if should_capture_context():
+            attributes._set_weave_item("client_version", version.VERSION)
+            attributes._set_weave_item("source", "python-sdk")
```

Comment:
I think we can always capture these.  Is there ever a case where this would be bad to capture?

Reply:
I split this into two separate flags. I agree that we would almost always want it for debugging, but I'm giving user the option under the theory that we should allow consistent configuration and give users control (esp. since they pay by usage)